src/jit/inline.cpp

```diff
@@ -578,8 +443,9 @@ void InlineContext::DumpData(Compiler* compiler, int indent)
     if (m_Parent == nullptr)
     {
         // Root method... cons up a policy so we can display the name
-        InlinePolicy* policy = InlinePolicy::GetPolicy(compiler, true);
-        printf("\nInlines [%u] into \"%s\" [%s]\n", compiler->fgInlinedCount, calleeName, policy->GetName());
+        InlinePolicy*   policy   = InlinePolicy::GetPolicy(compiler, true);
+        InlineStrategy* strategy = compiler->m_inlineStrategy;
```

Comment:
Yes, they're the same, and while not wrong, it is indeed confusing.

Thanks for catching this, I'll get it cleaned up.